tests/unit/models/Component.spec.js

```diff
@@ -410,19 +436,20 @@ describe('Test class: Component', () => {
         }));
 
         component.removeLinkAttribute('test', 'link');
-        expect(component.attributes).toEqual([new ComponentAttribute({
-          name: 'link',
-          value: ['other'],
-          type: 'String',
-          definition: new ComponentAttributeDefinition({
-            name: 'other',
-            type: 'Link',
-          }),
-        })]);
+        expect(component.attributes)
+          .toEqual([new ComponentAttribute({
+            name: 'link',
+            value: ['other'],
+            type: 'String',
+            definition: new ComponentAttributeDefinition({
+              name: 'other',
+              type: 'Link',
+            }),
+          })]);
       });
 
       it(
-        'Should remove value corresponding to the given id then the attribute if array is empty',
+        'Should remove value corresponding to the given id if the attribute if array is empty',
```

Comment:
Should remove value from given attribute then the attribute itself if it's empty.